torchao/experimental/ops/mps/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+python ../../kernels/mps/codegen/gen_metal_shader_lib.py
+
+export CMAKE_PREFIX_PATH=$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')
```

Comment:
why cmake prefix path points to site-packages?

Reply:
that's where the cmake stuff is. I am working on a conda environment.

Reply:
what do you mean thats where the cmake stuff is. prefix path is used for looking up packages https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html and other stuff